libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelectorAlternativeLocation.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, memo } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { withWidgetSettings } from '@shopgate/engage/core';
+import { Grid, ResponsiveContainer, SurroundPortals } from '@shopgate/engage/components';
+import { provideProductAlternativeLocation } from '../../action-creators';
+import { StockInfo } from '../StockInfo';
+import { locationName } from './FulfillmentSelectorLocation.style';
+import { itemColumn, itemSpacer } from './FulfillmentSelectorItem.style';
+import { PRODUCT_FULFILLMENT_SELECTOR_ALTERNATIVE_LOCATION } from '../../constants/Portals';
+import { getProductAlternativeLocations } from '../../selectors';
+import { SORT_CLOSEST_LOCATION_WITH_INVENTORY } from '../../constants';
+
+/**
+ * The FulfillmentSelectorLocation component
+ * @returns {JSX}
+ */
+function FulfillmentSelectorAlternativeLocation({
+  productId, show, alternativeLocations, provideAlternativeLocation, widgetSettings,
+}) {
+  const {
+    alternativeLocationOnPDP: {
+      show: widgetShow = false,
+      radius = 50,
```

Comment:
default value should be the value of the next admin setting

Reply:
The value from admin is not available in jsonp response. See merchant service JavaScriptS3.js